tests/core/gui/test_revoked_user.py

```diff
@@ -36,21 +36,28 @@ async def test_revoked_notification(
     ).dump_and_sign(alice_user_fs.device.signing_key)
     central_widget = logged_gui.test_get_central_widget()
     assert central_widget is not None
-    async with aqtbot.wait_signal(central_widget.new_notification):
+    async with aqtbot.wait_signal(central_widget.new_notification, timeout=3000):
         await alice_user_fs.backend_cmds.user_revoke(revoked_device_certificate)
-        await aqtbot.mouse_click(central_widget.menu.button_users, QtCore.Qt.LeftButton)
-        await aqtbot.mouse_click(central_widget.menu.button_devices, QtCore.Qt.LeftButton)
 
     # Assert dialog
     assert len(autoclose_dialog.dialogs) == 1
     dialog = autoclose_dialog.dialogs[0]
     assert dialog[0] == "Error"
     assert dialog[1] == "This device is revoked"
 
-    # Assert device and user widgets
-    devices_w = logged_gui.test_get_devices_widget()
     users_w = logged_gui.test_get_users_widget()
```

Comment:
I think we should use the brand new shinny `logged_gui.test_switch_to_devices_widget` (which really click on the page instead of just getting the widget)

Reply:
it's simplify things a lot ;)